doc/classes/SpatialMaterial.xml

```diff
@@ -221,7 +221,7 @@
 			If [code]true[/code], triplanar mapping is calculated in world space rather than object local space. See also [member uv1_triplanar].
 		</member>
 		<member name="metallic" type="float" setter="set_metallic" getter="get_metallic" default="0.0">
-			The reflectivity of the object's surface. The higher the value, the more light is reflected.
+			A high value makes the material appear more like a metal. Non-metals use [member albedo] as the diffuse color and add diffuse to the specular reflection. With non-metals, the reflection appears on top of [member albedo]. Metals use [member albedo] as a multiplier to the specular reflection and set the diffuse color to black resulting in a tinted reflection. Materials work better when fully metal or fully non-metal, values between [code]0[/code] and [code]1[/code] should only be used for blending between metal and non-metal sections. To alter the amount of reflection use [member roughness].
```

Comment:
There's no `albedo` member, it's either `albedo_color` or `albedo_texture`.

Reply:
Ah, right. its probably better to just say "albedo" anyway because it captures both